tests/conftest.py

```diff
@@ -6,22 +6,23 @@
 import pykorm
 
 
+class Score(pykorm.models.Nested):
+    exterior: int = pykorm.fields.DataField('exterior')
+    delicious: int = pykorm.fields.DataField('delicious', 10)
+
+
+class ScoreMixin(object):
+    score: Score = pykorm.fields.DictNestedField(Score, path=['spec', 'score'])
+
+
 @pykorm.k8s_custom_object('pykorm.infomaniak.com', 'v1', 'apples')
-class Apple(pykorm.ClusterModel):
+class Apple(ScoreMixin, pykorm.ClusterModel):
     variety: str = pykorm.fields.Spec('variety', 'default-variety')
-    tastyness: str = pykorm.fields.MetadataAnnotation('tastyness', 'very-tasty')
-    creationTimestamp: str = pykorm.fields.Metadata('creationTimestamp', readonly=True)
-
-    def __init__(self, name: str, variety: str):
-        self.name = name
-        self.variety = variety
 
 
 @pykorm.k8s_custom_object('pykorm.infomaniak.com', 'v1', 'peaches')
-class Peach(pykorm.NamespacedModel):
+class Peach(ScoreMixin, pykorm.NamespacedModel):
     variety: str = pykorm.fields.Spec('variety', 'default-variety')
-    price: str = pykorm.fields.Spec('price', 1)
```

Comment:
This makes test `tests/test_filter.py::test_filter_chain` fail. It was introduced in order to generate many Peach objects and test chain filters. We can either keep it or remove it but then we'd have to tweak the tests

Reply:
Ok, I have changed back.